tron/bin/action_runner.py

```diff
@@ -103,12 +103,19 @@ def parse_args():
     return parser.parse_args()
 
 
-def run_command(command):
+def run_command(command, run_id):
+    namespace, job, run_num, action = run_id.split('.')
```

Comment:
Can you turn this into a function that returns the env dictionary?

That will be easier to test and change in the future.

Reply:
This is too scary also. You are assuming there will only ever be 4 periods. Please make the function more defensive against this.

Also it needs to be in a big try/except. I don't want this split to cause a job not to run.

Reply:
Done.